### DISCUSSION TITLE
Accessing model data in repeater

### DISCUSSION THREAD

**Discussion by avinash403** [area17/twill#2333] [Q&A]
I have this parent model, let's say `Post` and child element with one to many relation let's say `Collaborator`. On my form, I have a list if collaborators added as a repeater to give functionality to create collaborators quickly. I want to put up a link to edit the collaborator on each of the repeater element. Is there a way to get the model data attached to a repeater element in the repeater blade file?

```
@twillRepeaterTitle('Collaborator')
@twillRepeaterTrigger('Add Collaborator')
@twillRepeaterGroup('app')

<x-twill::medias
    name="profile_picture"
    label="Profile Picture"
/>
<x-twill::input
    name="full_name"
    label="Full Name"
    :required="true"
/>

<a href="" {{-- here's where I need the model data to create the edit url --}}
   class="button button--action" style="float: right;">Edit</a>

```

**Comment by ifox** [comment]
@avinash403 I'm afraid this is currently not possible, however it would be straightforward to implement using an hidden field storing the id of the item that would then be dynamically used in JS to render the link you're looking for.
